game_engine/core/Statistics.cpp

```diff
@@ -50,18 +50,24 @@ void to_json(nlohmann::json &json, const PlayerStatistics &stats) {
             FIELD_TO_JSON(random_id),
             FIELD_TO_JSON(rank),
             FIELD_TO_JSON(last_turn_alive),
+            FIELD_TO_JSON(last_turn_ship_spawn),
             {"final_production", final_production},
             FIELD_TO_JSON(total_production),
             FIELD_TO_JSON(max_entity_distance),
             FIELD_TO_JSON(number_dropoffs),
             FIELD_TO_JSON(interaction_opportunities),
             FIELD_TO_JSON(ships_captured),
             FIELD_TO_JSON(ships_given),
+            FIELD_TO_JSON(ships_spawned),
```

Comment:
While we're at it, we should stop writing unused fields - `ships_captured`, `ships_given`, `total_mined_from_captured`, and perhaps `interaction_opportunities`.

Reply:
I actually specifically advised against that on discord. But after thinking about it some more the risk seems very minimal and while the benefit is also fairly small it probably is good on balance.

Since you're obviously in favor, I'm fine with them going as well.